src/routes/users.rs

```diff
@@ -23,25 +21,20 @@ struct NewUserData {
 
 #[post("/users", format = "application/json", data = "<new_user>")]
 pub fn post_users(new_user: Json<NewUser>, conn: db::Conn) -> Result<Json<JsonValue>, Errors> {
-    use crate::schema::users;
-
+    use db::users::{email_exists, username_exists};
```

Comment:
Please move this to the beginning of the file.

Reply:
Yes, I agree, thank you